about.html

```diff
@@ -0,0 +1,101 @@
+                    <a class="nav-item" href="">Join</a>
+                    <a class="nav-item" href="">Sponsor</a>
+                    <a class="nav-item nav-item1" href="">Sign in</a>
+            </div>
+        </nav>
+    </header>
+    <section class="Home">
+        <div class="page">
+            <div class="align"> <h2 class="greeting">Hello dev world!</h2>
+            </div>           
+           <div class="align2"> <h1 class="subject">Google DevFest 2022</h1></div>
+            <div class="align3"><p class="details1">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Architecto pariatur officiis quaerat culpa quasi, repellat possimus tempore esse porro suscipit eaque, iste, soluta ad tempora! Reiciendis neque totam cum eius?
+                                </p>
+            </div>
+           <div class="align4"> <p class="time">For more information, visit our website at www.google developer group.com</p></div>
+        </div>
+     </div>
+        <div class="about-logo">
+           <div class="style">
+            <h2 class="subject2">Google DevFest 2022</h2>
+            <hr class="logo-hor">
+           </div>
+            <p class="logo-paragraph">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente totam tempore ex repellat.
+            </p>
+            <div class="logo2">
+                <img class="logo-google" src="./img/GDG.png" alt="">
+            </div>
+            <p class="logo-paragraph">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Tenetur obcaecati sapiente, vero es.
+            </p>
+            <hr class="image-hr">
+        </div>  
+        <div class="past-events">
+            <div class="seep">
+                <div class="see-heading">See past GDG events</div>
+            <hr class="seepast-hr">
+            </div>
+        <div class="see-imgs">
+            <div class="picture1">
+                <img class="pic1" src="./img/download.jfif" alt="">
+                <div class="centered">2010</div>
+            </div>
+        <div class="picture2">
+            <img class="pic2" src="./img/FEST1.jfif" alt="">
+            <div class="centered">2013</div>
+         </div>
+          </div>
+        </div>
+       <div class="two-sections">
+     <section class="partners">
+        <div class="line">
+          <h3 class="partners1">Partners</h3>
+          <hor class="hor2"></hor>
+        </div>
+        <ul class="list">
+            <li><img class="li" src="./img/mozilla.png" alt=""></li>
+            <li><img class="li kakao" src="./img/kakao.png" alt=""></li>
+            <li><img class="li" src="./img/naver.png" alt=""></li>
+        </ul>
+     </section>
+     <footer class="footer">
+     <div class="end">
+        <img class= end-img src="./img/GDG.png" alt="">
+        <h3 class="camp">Google Dev Group.org</h3>
+     </div>
+     <p class="copyright">©2022</p>
+     </footer>
+    </div>
+</section>
```

Comment:
- [ ] Please you are required to have only the logo and a small note in the about page footer, not partners please kindly fix it 

<img width="1425" alt="image" src="https://user-images.githubusercontent.com/37901469/192102935-af87e424-88cb-4d47-a602-65b1b1c759db.png">